toolkit/scripts/toolchain.mk

```diff
@@ -19,6 +19,8 @@ populated_toolchain_chroot = $(toolchain_build_dir)/populated_toolchain
 toolchain_sources_dir = $(populated_toolchain_chroot)/usr/src/mariner/SOURCES
 populated_toolchain_rpms = $(populated_toolchain_chroot)/usr/src/mariner/RPMS
 toolchain_spec_list = $(toolchain_build_dir)/toolchain_specs.txt
+toolchain_actual_conents = $(toolchain_build_dir)/actual_archive_contents.txt
```

Comment:
Should this be "toolchain_actual_contents"?

Reply:
Fixed in the makefile fixup PR